multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/api/impl/FilesApiServiceImpl.java

```diff
@@ -424,13 +433,22 @@ private HttpResponse<InputStream> callRemoteEndpointWithRetry(HttpClient client,
                                                                UriUtil.stripUserInfo(decodedUrl));
                     throw new SLException(errorMessage);
                 }
-                throw new SLException(MessageFormat.format(Messages.ERROR_FROM_REMOTE_MTAR_ENDPOINT, request.uri(), response.statusCode(),
-                                                           error));
+                throw new SLException(MessageFormat.format(Messages.ERROR_FROM_REMOTE_MTAR_ENDPOINT, getUriFromAtSign(request.uri()),
+                                                           response.statusCode(), error));
             }
             return response;
         });
     }
 
+    private String getUriFromAtSign(URI uri) {
```

Comment:
Maybe rename the function to getMaskedUri, append "..." in front and return the string without "@", e.g. "...google.com" instead of "@google.com"